qos-service-api/src/main/java/com/palantir/atlasdb/qos/ratelimit/RateLimitExceededException.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2017 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the BSD-3 License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://opensource.org/licenses/BSD-3-Clause
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.atlasdb.qos.ratelimit;
+
+public class RateLimitExceededException extends RuntimeException {
```

Comment:
nit: add some docs here?

Reply:
Ack, will do after the ongoing superpriority ticket